### PR TITLE
Update dependencies and README info

### LINKED ISSUE #8: ModuleNotFoundError: No module named 'pandas'
working with latest devel, I get this error when I try to run the code. I installed with `pip install -e .`

### PR DESCRIPTION
**Pull request by vmharvey**
Fixes #8 

Also adds instructions to README about how to install.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ Goals:
 5. Bonus: Candidate preferences for roles
 6. Pass pylint
 
+## Installation for developers
+
+1. Clone the repo
+2. Run `pip install -e .` to install all dependencies with pip.
```
Comment (ibarnsley) [inline README.md:17]: Nice addition!

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+pandas
```
Comment (ibarnsley) [inline requirements.txt:1]: Looks good!

**Review by ibarnsley — APPROVED** [review]
Looks good to me!

**Comment by vmharvey** [comment]
woo!

**Pull request merged**